src/org/zaproxy/zap/extension/frontEndScanner/ExtensionFrontEndScanner.java

```diff
@@ -0,0 +1,328 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.

```

Comment:
Should use std header with the date: https://github.com/zaproxy/zap-admin/blob/master/docs/headers/license.java

Reply:
Done!